src/Raven.Client/Http/ServerNode.cs

```diff
@@ -12,6 +12,7 @@ public enum Role
             None = 0,
             Promotable = 1,
             Member = 2,
+            Witness = 3,
```

Comment:
You are going to need a promotable version that will go to witness, not a member as well

When you add a new witness node to a cluster, it cannot be immediately added, you have to sync its state first.